depthai_sdk/docs/source/blob_nnet_manager.rst

```diff
@@ -0,0 +1,76 @@
+======================
+Blob and Nnet managers
+======================
+
+Blob manager
+------------
+
+``Blob manager`` is a class that is made to help you with downloading neural networks as MyriadX blobs.
+
+Getting started
+^^^^^^^^^^^^^^^
+
+``Blob manager`` is very easy and straight forward to use. We declare it and pass which project we want to use as it's argument.
+
+.. literalinclude:: ./examples/code_fractions/blob.py
+   :language: python
+   :linenos:
+
+After that, the ``blob`` is stored in our variable and we can then pass it to our ``Nnet manager``, as we will se bellow.
+Of course the ``face-detection-retail-0004`` is not the only project that we can use. All other projects that are supported are:
+
+.. include::  blob_projects.rst
```

Comment:
@ZigiMigi BlobManager supports all models in both [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo) and [our model zoo](https://github.com/luxonis/depthai-model-zoo). 
By using `configPath` you can download any arbitrary model and `zooDir` param you can specify a path to a custom model zoo as well (which can contain both already compiled blobs and model yml files)

BlobManager is able to both reuse existing blobs, download models from model zoo and compile custom models based on yml config